debian/preview/Dockerfile

```
@@ -22,7 +22,7 @@ RUN set -x; apt-get update \
   && JAVA_MAJOR_VERSION=$(echo $JAVA_VERSION | cut -d'+' -f1) \
```

```suggestion
  && JAVA_MAJOR_VERSION=$(echo $JAVA_VERSION | cut -d'+' -f1 | cut -d'.' -f1) \
```

WDYT of this change to ensure the `JAVA_MAJOR_VERSION` is always the first major digit (wether there are dots or not or '+' or not).

I've used this trick in https://github.com/jenkins-infra/packer-images/pull/860/files#diff-485e38775f798b1a8181ceada9148f140b6487f8bcadc39867338df791d06ca6R304-R307 FWIW.

- It would avoid the hardcoding
- It ensure that all occurence using `JAVA_MAJOR_VERSION` (there are two...) would work.